### PR TITLE
PSMDB-929 Release notes for 4.2.17-17

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/conf.py
	modified:   source/data_at_rest_encryption.rst
	new file:   source/release_notes/4.2.17-17.rst
	modified:   source/release_notes/index.rst

### CODE REVIEW

File: source/release_notes/4.2.17-17.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* The number of modify updates in WiredTiger is now limited to prevent performance issues
+* Errors related to proxy connections receive new error codes which don't change the behavior of clients' drivers.
+
+Find the full list of changes in the  `MongoDB 4.2.16 Community Edition <https://docs.mongodb.com/manual/release-notes/4.2/#4.2.16---sep-13--2021>`_ and MongoDB 4.2.17 Community Edition <>`_ release notes.
+
+Percona Server for MongoDB Release Highlights
+=============================================
+
+* The ``mongod`` user is not deleted during the re-installation of Percona Server for MongoDB on RPM-based operating systems. This preserves the permissions to the MongoDB data directory for the ``mongod`` user as its user ID and group ID remain unchanged.
+* RPM packages now include the ``cyrus-sasl`` modules to enable LDAP authentication via native LDAP against Active Directory
```
Comment (spron-in) [inline source/release_notes/4.2.17-17.rst]: I suggest to keep only one Release Highlights section. Our goal is to deliver open source software to community and customers and if MongoDB corp team did most of the job - so be it :)

I feel that our improvement and a bug fix are not that significant for highlights. So in here it should be okay to keep only the ones which came from MongoDB corp, and our tickets should be okay to be under corresponding sections. 

I really like the value oriented description that you created in Release Highlights, we might reuse it in Improvements and Bugs Fixed sections. What do you think?

Comment (nastena1606) [reply source/release_notes/4.2.17-17.rst]: Fixed: removed MongoDB highlights section, making it the only one. 
Extended PSMDB Jira issue subjects

**Comment by kwilktx** [comment]
Made a few additions to the 4.2.x highlights section in the doc on the g-Drive - here:
https://docs.google.com/document/d/1Sp0_jLGmf26uNvBSMtgIQmN9RFXfi4PPiwFCRDJlLW8/edit#

File: .netlify/state.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+	"siteId": "af1034bc-c9f2-4d8a-b0a3-a7cca4264af5"
+}
```
Comment (igorsol) [inline .netlify/state.json]: PLease add newline at the end of file

**Review by igorsol — CHANGES_REQUESTED** [review]

**Review by sandraromanchenko — APPROVED** [review]

File: .netlify/state.json (hunk 2 above)
Comment (nastena1606) [reply .netlify/state.json]: This file is auto-generated. Updated the .gitignore to not track it

**Pull request merged**